convert/src/vertex_to_bd/vertex_to_badgerdoc_use_case.py

```diff
@@ -1,53 +1,72 @@
 import tempfile
 from pathlib import Path
+from uuid import uuid4
 
-from ..config import DEFAULT_PAGE_BORDER_OFFSET
-from .badgerdoc_format.annotation_converter import AnnotationConverter
-from .badgerdoc_format.badgerdoc_format import BadgerdocFormat
-from .badgerdoc_format.pdf_renderer import PDFRenderer
-from .badgerdoc_format.plain_text_converter import (
+import requests
+from tenant_dependency import TenantData
+
+from src.config import DEFAULT_PAGE_BORDER_OFFSET, settings
+from src.logger import get_logger
+from src.vertex_to_bd.badgerdoc_format.annotation_converter import (
+    AnnotationConverter,
+)
+from src.vertex_to_bd.badgerdoc_format.badgerdoc_format import BadgerdocFormat
+from src.vertex_to_bd.badgerdoc_format.pdf_renderer import PDFRenderer
+from src.vertex_to_bd.badgerdoc_format.plain_text_converter import (
     TextToBadgerdocTokensConverter,
 )
-from .models.vertex_models import LabelStudioModel, S3Path
+from src.vertex_to_bd.models.vertex_models import LabelStudioModel, S3Path
+
+LOGGER = get_logger(__file__)
 
 
 class VertexToBDConvertUseCase:
     def __init__(
         self,
         s3_client,
+        current_tenant: str,
```

Comment:
personally I do not understand the refactoring
Why overcomplicate the object with a lot of attributes?

As I see for now we use it in only one API call.

In my taste that is too many attributes for one object.

Reply:
As soon as we create LabelStudioToBDConvertUseCase instance and use it, I pass all the relevant data needed by its inner logic. This class became larger - it communicates with other microservices and needs data like current_tenant and jw-token to make requests.